wasabi/printer.py

```diff
@@ -92,14 +93,36 @@ def good(
             title, text, style=MESSAGES.GOOD, show=show, spaced=spaced, exits=exits
         )
 
+    @overload
     def fail(
         self,
         title: Any = "",
         text: Any = "",
         show: bool = True,
         spaced: bool = False,
-        exits: Optional[int] = None,
+        exits: Optional[Literal[0, False]] = None,
     ):
```

Comment:
```suggestion
    ): -> Union[str, None]
```

Reply:
So for some reason, adding this type annotation gives a mypy error:

```
Overloaded function signatures 1 and 2 overlap with incompatible return types  [misc]
```

and I'm not sure why. If I leave that off, the error is not raised and type annotations work like expected still when calling the `Printer.fail` method